app/src/main/java/chat/rocket/android/service/RealmBasedConnectivityManager.java

```diff
@@ -163,8 +165,8 @@ public void notifyConnecting(String hostname) {
       }
 
       return connectToServer(hostname)
-          //.doOnError(RCLog::e)
-          .retryWhen(RxHelper.exponentialBackoff(3, 500, TimeUnit.MILLISECONDS));
+          .doOnError(RCLog::e)
+          .retryWhen(RxHelper.exponentialBackoff(Integer.MAX_VALUE, 500, TimeUnit.MILLISECONDS));
```

Comment:
Also I think on putting a counter there too, just like the web client, specially because we're using an exponential backoff retry time there, so it might take a while to try reconnecting again as time passes.